src/Apps/Order/Routes/Shipping2/Hooks/useParseOrderData.ts

```diff
@@ -73,38 +96,26 @@ export const useParseOrderData = (props: ShippingProps): ParsedOrderData => {
   )
 
   const shippingQuotes = extractNodes(firstLineItem.shippingQuoteOptions) ?? []
-  const selectedShippingQuoteId = shippingQuotes.find(quote => quote.isSelected)
-    ?.id
+  const selectedShippingQuoteId =
+    shippingQuotes.find(quote => quote.isSelected)?.id ?? null
 
   return {
-    savedFulfillmentData,
+    savedFulfillmentDetails: savedFulfillmentData,
```

Comment:
1:2 -- is it `savedFulfillmentDetails` or `savedFulfillmentData`? Pick one, use it all the way up and down